lms/djangoapps/courseware/access_utils.py

```diff
@@ -75,7 +75,7 @@ def check_start_date(user, days_early_for_beta, start, course_key, display_error
     Returns:
         AccessResponse: Either ACCESS_GRANTED or StartDateError.
     """
-    start_dates_disabled = settings.FEATURES['DISABLE_START_DATES']
+    start_dates_disabled = settings.FEATURES.get('DISABLE_START_DATES', False)
```

Comment:
It feels safer to set it to false in case there are no settings specified for it. But I have just used a get now instead of passing value to it. Hope it's okay :)